internal/dag/policy.go

```diff
@@ -25,15 +26,28 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation"
 )
 
+func retryOn(ro []projcontour.RetryOn) string {
```

Comment:
Nit: could you add a comment mentioning that the CRD validation ensures that only valid retry strings are present in this slice?